src/main/java/seedu/address/logic/parser/property/FilterPropertiesCommandParser.java

```diff
@@ -53,12 +53,18 @@ public FilterPropertiesCommand parse(String args) throws ParseException {
 
         if (argMultimap.getValue(PREFIX_PRICE_RANGE).isPresent()) {
             PriceRange priceRange = ParserUtil.parsePriceRange(argMultimap.getValue(PREFIX_PRICE_RANGE).get());
+            if (priceRange.isReset()) {
+                throw new ParseException("If -r flag is used, it cannot be empty.");
```

Comment:
Would this make more sense? If the error is thrown, the -r is def used right?
```suggestion
                throw new ParseException("-r flag should not be left empty.");
```
Same comment for the -c flag!

Reply:
Oh i thought maybe the user would be confused since -r flag is optional, but on second thought i don't think anybody would be confused hahah